rewrite-gradle/src/test/java/org/openrewrite/gradle/ChangeDependencyTest.java

```diff
@@ -15,8 +15,10 @@
  */
 package org.openrewrite.gradle;
 
+import org.junit.jupiter.api.DisplayName;
```

Comment:
I think these might be left over from an earlier change perhaps?
```suggestion
```

Reply:
Sorry, fixed